git_machete/tests/functional/test_machete.py

```diff
@@ -1790,10 +1790,23 @@ def test_github_create_pr(self) -> None:
             .push()
         )
         self.launch_command('discover')
-        self.assert_command(['github', 'create-pr'], "All commits in `testing/endpoints` branch  are already included in `develop` branch.\nCannot create pull request.\n", strip_indentation=False)
+
+        machete_client = MacheteClient(cli_opts, git)
+        machete_client.read_definition_file()
+        expected_error_message = "All commits in `testing/endpoints` branch  are already included in `develop` branch.\nCannot create pull request."
```

Comment:
Oh I see why that `print(e)` was needed... anyway, the new approach is better (more consistent with how exceptions are handled everywhere else)